srcpkgs/rofi-wayland/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'rofi-wayland'
+pkgname=rofi-wayland
+version=1.7.5+wayland1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config git flex glib-devel wayland-devel"
+makedepends="libglib-devel cairo-devel pango-devel libxkbcommon-devel
+ gdk-pixbuf-devel wayland-devel libmpdclient-devel libnl3-devel libxcb-devel
+ xcb-util-devel xcb-util-wm-devel xcb-util-cursor-devel
+ startup-notification-devel wayland-protocols alsa-lib-devel check-devel"
+depends="rofi>=${version%+*} bash which sed"
```

Comment:
why does this fork depend on the original rofi?

Reply:
This package only provides two binaries (rofi-wayland and rofi-theme-selector-wayland) and their corresponding .desktop, icon, and man pages. Everything else needed and related to rofi is provided by the main rofi package.

Reply:
i.e. the mentality behind this package is that this package complements the main rofi package. It only provides what isn't already there in the main rofi package.

Reply:
I see, do you still need to run rofi-wayland instead of rofi when you want it to use the rofi with wayland support? If that's the case, then scripts that call rofi will just call the xorg rofi...

Reply:
> scripts that call rofi will just call the xorg rofi


Good point... I hadn't thought about that at all.....

(imma draft this PR now and think about this later. Got an exam coming up :slightly_smiling_face:)

Reply:
I guess I could look into xbps-alternatives